DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -514,6 +514,20 @@
   <class name="edm::Wrapper< std::vector<edm::Ptr<pat::Jet> > >" />
   <class name="edm::Wrapper< std::vector< std::vector<edm::Ptr<pat::Jet> > > >" />
 
+  <class name="pat::PATTauDiscriminatorBase">
+    <field name="transientVector_" transient="true"/>
+  </class>  
+  <class name="pat::PATTauDiscriminator"  ClassVersion="3">
+   <version ClassVersion="3" checksum="2916098408"/>  
+  </class>  
```

Comment:
Hi guys,

could it also be that by the contraction of the template base class to a concrete base class (e.g. for pat::Tau) in the typedef the derived class will also be treated as a non template class and the whole syntax is OK? 

I'm still missing what I should do about the raised issue. Just let me know whether I should make an effort to introduce a Class_Version class as proposed by Chris to the base class and or to the derived class and what version I should give them to start with? If so the questions remains whether the same should done with the analogue classes in DataFormats/TauReco?

Given that this is a subject that still needs general clarification and that if it is an issue it also is for existing code in DataFormats/TauReco would it make sense to raise it to an issue that should just be resolved in a general effort maybe independently from the pre12 deadline?

Once clarification has been reached what to do here I'm fine to subscribe for it, but I'm afraid this wont happen before next week. 

Cheers,
Roger